ci/pip_requirements.txt

```diff
@@ -3,7 +3,7 @@ coveralls
 natsort
 lockfile
 CacheControl
-Sphinx
+Sphinx <= 3.0
```

Comment:
Can you add a note to the ChangeLog about pinning down the Sphinx version?

Reply:
Sure!